src/polymer/iron-flex-layout-classes.ts

```diff
@@ -0,0 +1,262 @@
+      range: sourceRangeForPrependContent(parsedDocument, body)
+    }];
+    warnings.push(warning);
+  }
+}
+
+function getMissingStyleModules(node: dom5.Node) {
+  const usedModules = styleModules.filter((m) => !!dom5.query(node, m.selector))
+                          .map((m) => m.name);
+  if (!usedModules.length) {
+    return;
+  }
+  const styleNodes = dom5.queryAll(node, isStyleInclude);
+  let currentModules: string[] = [];
+  for (const style of styleNodes) {
+    currentModules =
+        [...currentModules, ...dom5.getAttribute(style, 'include')!.split(' ')];
+  }
+  const missingModules =
+      usedModules.filter((m) => currentModules.indexOf(m) === -1);
+  if (missingModules.length) {
+    return missingModules.join(' ');
+  }
+}
+
+function createWarning(
+    parsedDocument: ParsedHtmlDocument,
+    node: dom5.Node,
+    missingModules: string) {
+  const multi = missingModules.indexOf(' ') > -1;
+  return new FixableWarning({
+    code: 'iron-flex-layout-classes',
+    message: `Style module${multi ? 's are' : ' is'} used but not imported:
+
+  ${missingModules}
+
+Import ${multi ? 'them' : 'it'} in the template style include.`,
+    parsedDocument,
+    severity: Severity.WARNING,
+    sourceRange: parsedDocument.sourceRangeForStartTag(node)!
+  });
+}
+
+function getStyleNodeToEdit(node: dom5.Node) {
+  let styleToEdit = null;
+  for (const style of dom5.queryAll(node, isStyleInclude)) {
+    // Get the first one of the styles with include attribute, otherwise
+    // prefer styles that already include iron-flex-layout modules.
+    if (!styleToEdit ||
+        styleModulesRegex.test(dom5.getAttribute(style, 'include')!)) {
+      styleToEdit = style;
+    }
+  }
+  // Fallback to style without include attribute.
+  return styleToEdit || dom5.query(node, p.hasTagName('style'));
+}
+
+function sourceRangeForAppendAttributeValue(
+    parsedDocument: ParsedHtmlDocument, node: dom5.Node, attr: string) {
+  const tagRange = parsedDocument.sourceRangeForAttributeValue(node, attr)!;
+  return {
+    file: tagRange.file,
+    start: {line: tagRange.end.line, column: tagRange.end.column - 1},
+    end: {line: tagRange.end.line, column: tagRange.end.column - 1}
+  };
+}
+
+function sourceRangeForAddAttribute(
+    parsedDocument: ParsedHtmlDocument, node: dom5.Node) {
+  const tagRange = parsedDocument.sourceRangeForStartTag(node)!;
+  return {
+    file: tagRange.file,
+    start: {line: tagRange.end.line, column: tagRange.end.column - 1},
+    end: {line: tagRange.end.line, column: tagRange.end.column - 1}
+  };
+}
+
+function sourceRangeForPrependContent(
+    parsedDocument: ParsedHtmlDocument, node: dom5.Node) {
+  const tagRange = parsedDocument.sourceRangeForStartTag(node)!;
+  return {
+    file: tagRange.file,
+    start: {line: tagRange.end.line, column: tagRange.end.column},
+    end: {line: tagRange.end.line, column: tagRange.end.column}
+  };
+}
+
+registry.register(new IronFlexLayoutClasses());
```

Comment:
I think this line should be at the bottom of the file.

Reply:
Done